packages/App/src/components/ButtonApp.tsx

```diff
@@ -1,30 +1,39 @@
 import React, {FC} from 'react';
-import {TouchableOpacity, Text, StyleSheet, Image} from 'react-native';
+import {
+  TouchableOpacity,
+  Text,
+  StyleSheet,
+  Image,
+  ImageSourcePropType,
+} from 'react-native';
+import addTransactionSource from '../../pictures/balance/income.png';
+import confirmSource from '../../pictures/balance/basic-tick.png';
+import walletSource from '../../pictures/balance/wallet.png';
+import plusSource from '../../pictures/debt/plus.png';
+import minusSource from '../../pictures/debt/minus.png';
 
 interface Props {
   disabled?: boolean;
   label: string;
-  styleBtn?: any;
-  styleTxt?: any;
   onPress?: () => void;
-  image?: any;
+  image?: string;
```

Comment:
Why not to accept `ImageSourcePropType` why do we need to map it?

Reply:
I tried it on the first place, but I don`t know why source got "red underline" and was written that ImageSourcePropType doesn`t match. I think it`s happened cuz I sent link as a prop, so I attempted to avoid by this way)